bitcoin/src/taproot.rs

```diff
@@ -10,15 +10,17 @@ use core::convert::TryFrom;
 use core::fmt;
 
 use bitcoin_internals::write_err;
-use secp256k1::{self, Scalar, Secp256k1};
+use secp256k1::{self, Scalar};
 
 use crate::consensus::Encodable;
-use crate::crypto::key::{TapTweak, TweakedPublicKey, UntweakedPublicKey, XOnlyPublicKey};
 use crate::hashes::{sha256t_hash_newtype, Hash, HashEngine};
 use crate::prelude::*;
 use crate::{io, Script, ScriptBuf};
 
+pub use secp256k1::{constants, Secp256k1, KeyPair, XOnlyPublicKey, Verification, Parity};
+pub use crate::crypto::key::{TapTweak, TweakedPublicKey, UntweakedPublicKey};
```

Comment:
We are missing a few other items from `key`: `SortKey`, `Error`, `SecretKey`, `PublicKey`, `TweakedKeypair`, `UntweakedKeypair`. These are all `pub` within the `key` module but I don't see where the `key` module itself is exported.

Reply:
We should export `key` at root, definitely not in `taproot` which is unrelated.

Reply:
Its exported from the root already. Will change this PR as suggested and raise a new discussion item to address the re-export thing I was trying to achieve here, thanks.